tests/hap/test_svm_wfc3ir.py

```diff
@@ -0,0 +1,213 @@
+    manifest_filename = "_".join(tokens_tuple)
+    print("\nconstruct_manifest_filename. Manifest filename: {}".format(manifest_filename))
+
+    return manifest_filename
+
+
+@pytest.fixture(scope="module", autouse=True)
+def svm_setup(gather_data_for_processing):
+    # Act: Process the input data by executing runsinglehap - time consuming activity
+
+    current_dt = datetime.datetime.now()
+    print(str(current_dt))
+    print("\nsvm_setup fixture")
+
+    # Read the "poller file" and download the input files, as necessary
+    input_names = gather_data_for_processing
+
+    # Run the SVM processing
+    path = os.path.join(os.path.dirname(__file__), POLLER_FILE)
+    try:
+        status = runsinglehap.perform(path)
+
+    # Catch anything that happens and report it.  This is meant to catch unexpected errors and
+    # generate sufficient output exception information so algorithmic problems can be addressed.
+    except Exception as except_details:
+        print(except_details)
+        pytest.fail("\nsvm_setup. Exception Visit: {}\n", path)
+
+    current_dt = datetime.datetime.now()
+    print(str(current_dt))
+
+
+# TESTS
+
+def test_svm_manifest_name(construct_manifest_filename):
+    # Construct the manifest filename from the header of an input file in the list and check it exists.
+    path = Path(construct_manifest_filename)
+    print("\ntest_svm_manifest. Filename: {}".format(path))
+
+    # Ensure the manifest file uses the proper naming convention
+    assert (path.is_file())
+
+
+def test_svm_wcs(gather_output_data):
+    # Check the output primary WCSNAME includes FIT_SVM_GAIA as part of the string value
+    tdp_files = [files for files in gather_output_data if
+                 files.lower().find("total") > -1 and files.lower().endswith(".fits")]
+
+    for tdp in tdp_files:
+        wcsname = fits.getval(tdp, "WCSNAME", ext=1).upper()
+        print("\ntest_svm_wcs.  WCSNAME: {} Output file: {}".format(wcsname, tdp))
+        assert WCS_SUB_NAME in wcsname, f"WCSNAME is not as expected for file {tdp}."
+
+
+def test_svm_empty_cats(gather_output_data):
+    # Check the output catalogs should contain > 0 measured sources
+    cat_files = [files for files in gather_output_data if files.lower().endswith("-cat.ecsv")]
+
+    valid_tables = {}
+    for cat in cat_files:
+        table_length = len(ascii.read(cat, format="ecsv"))
+        print("\ntest_svm_cat_sources. Number of sources in catalog {} is {}.".format(cat, table_length))
+        valid_tables[cat] = table_length > 0
+    bad_tables = [cat for cat in cat_files if not valid_tables[cat]]
+    assert len(bad_tables) == 0, f"Catalog file(s) {bad_tables} is/are unexpectedly empty"
+
+def test_svm_point_cats(gather_output_data):
+    # Check that the point catalogs have the expected number of sources
+    cat_files = [files for files in gather_output_data if files.lower().endswith("point-cat.ecsv")]
+    expected_sources = {'ir': 358}
```

Comment:
Just a suggestion for this and the next test as this is developer preference.  Putting expected_sources here keeps the values with the test, or putting the variable at the top of the module keeps the  variable which likely to change as algorithms evolve in one handy location.

Reply:
They have been renamed (to unique values) and moved to the top of the module in 2893c91.  This does make it easier to maintain these tests.